src/impl_x86__base_implementation.inl

```diff
@@ -667,6 +667,9 @@ X86Microarchitecture GetX86Microarchitecture(const X86Info* info) {
       case CPUID(0x12, 0x01):
         // https://www.amd.com/system/files/TechDocs/44739_12h_Rev_Gd.pdf
         return AMD_K12;
+      case CPUID(0x12, 0x00):
+        // https://www.amd.com/system/files/TechDocs/44739_12h_Rev_Gd.pdf
+        return AMD_LLANO;
```

Comment:
> Does it have to be merged with K12?
I'd leave it as `AMD_LLANO` since that's an accurate definition and we have a lot of cases at Intel so that's fine.